content/sensu-go/5.21/guides/monitor-sensu.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Monitor Sensu with Sensu"
+linkTitle: "Monitor Sensu with Sensu"
+description: "Make sure your Sensu components are properly monitored. This guide describes best practices and strategies for monitoring Sensu."
+weight: 230
+version: "5.21"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.21:
+    parent: guides
+---
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
+- [Monitor your Sensu dashboard instances](#monitor-your-sensu-dashboard-instances)
+- [Monitor your Embedded etcd](#monitor-your-embedded-etcd)
+- [Monitor your External etcd](#monitor-your-external-etcd)
+- [PostgreSQL PLACEHOLDER](#postgresql-placeholder)
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu.
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard.
```

Comment:
But why monitor Sensu? Chatted w/ @cwjohnston and we need to have a narrative around why a customer would want to monitor Sensu in the first place.